config/vufind/Blender.ini

```diff
@@ -0,0 +1,231 @@
+genre_facet        = Genre
+era_facet          = Era
+geographic_facet   = Region
+publishDate        = "adv_search_year"  ; share year string w/advanced search page
+
+[FacetLabels]
+labelSections[] = Advanced_Facets
+labelSections[] = HomePage_Facets
+labelSections[] = ResultsTop
+labelSections[] = Results
+labelSections[] = ExtraFacetLabels
+checkboxSections[] = CheckboxFacets
+
+; This setting lists configuration settings defining checkbox facets. If you use
+; a custom section to configure additional facets, be sure to add it to this list
+; so labels display correctly in history, the advanced search editor, etc.
+checkboxSections[] = CheckboxFacets
+
+; This section is used to identify facets for special treatment by the SideFacets
+; recommendations module.
+[SpecialFacets]
+; Any fields listed below will be treated as year-based date ranges rather than plain
+; facets:
+dateRange[] = publishDate
+; Any fields listed below will be treated as year/month/day-based date ranges rather
+; than plain facets:
+;fullDateRange[] = example_field_date
+; Any fields listed below will be treated as numeric ranges rather than plain facets:
+;numericRange[] = example_field_str
+; Any fields listed below will be treated as free-form ranges rather than plain
+; facets:
+;genericRange[] = example_field_str
+; Any fields listed below will be treated as hierarchical facets
+; (see https://wiki.apache.org/solr/HierarchicalFaceting but note that we always
+; use a trailing slash to avoid ambiquities)
+;hierarchical[] = building
+;hierarchical[] = format
+; Sort options for hierarchical facets:
+; How hierarchical facets are sorted. Default is result count, but alternative ways
+; can be specified:
+; top = Sort the top level list alphabetically, others by result count (useful e.g.
+;       for a large number of building facets where top level is organization and
+;       second level the library branch)
+; all = Sort all levels alphabetically
+hierarchicalFacetSortOptions[building] = top
+; How hierarchical facet values are displayed in the records:
+; single = Display only the deepest level (default)
+; full   = Display full hierarchy for each entry
+;hierarchicalFacetDisplayStyles[format] = full
+; Separator used when display hierarchical facet levels (default is "/")
+;hierarchicalFacetSeparators[format] = " > "
+
+; This section is reserved for special boolean facets.  These are displayed
+; as checkboxes.  If the box is checked, the filter on the left side of the
+; equal sign is applied.  If the box is not checked, the filter is not applied.
+; The value on the right side of the equal sign is the text to display to the
+; user.  It will be run through the translation code, so be sure to update the
+; language files appropriately.
+;
+; Leave the section empty if you do not need checkbox facets.
+[CheckboxFacets]
+; blender_backend facet is special and allows limiting of results to one of the
+; actual backends:
+blender_backend:Solr = "Local Index"
```

Comment:
This seems like another potential use case for the recently-requested radio button facets (see [VUFIND-1526](https://vufind.org/jira/browse/VUFIND-1526))

Reply:
Otherwise yes, but you can select e.g. two out of three.